src/about-us/index.html

```diff
@@ -0,0 +1,181 @@
+        </p>
+        <ul class="list list__links">
+            <li class="list_item">
+                <a href="/the-bureau/" class="list_link">
+                    Learn more about the Bureau
+                </a>
+            </li>
+            <li class="list_item">
+                {# TODO: Add link to Consumer Guides #}
+                <a class="list_link u-link__disabled">
+                    Find resources and tools for consumers
+                </a>
+            </li>
+            <li class="list_item">
+                <a href="/offices/payments-to-harmed-consumers/" class="list_link">
+                    Find out about our work to protect consumers
+                </a>
+            </li>
+        </ul>
+    </section>
+    <div class="content-l">
+        {# The extra div is to prevent border tapering from combining block__border and content-l classes #}
+        <div class="content-l_col-1-2">
+            <section class="block
+                        block__border-top
+                        block__padded-top
+                        block__flush-bottom">
+                <h2>CFPB Leadership</h2>
+                <div class="media bureau-bio u-mb30">
+                    <div class="media_image-container">
+                        <img class="media_image" src="/static/img/cordray_round_360.jpg">
+                    </div>
+                    <div class="media_body">
+                        <span class="h6">Director</span>
+                        <h3 class="h2 bureau-bio_name">Richard Cordray</h3>
+                        <a class="jump-link jump-link__right" href="about-rich-cordray">
+                            Cordray's bio
+                        </a>
+                    </div>
+                </div>
+                <div class="media bureau-bio u-mb30">
+                    <div class="media_image-container">
+                        <img class="media_image" src="/static/img/antonakes_round_360.jpg">
+                    </div>
+                    <div class="media_body">
+                        <span class="h6">Deputy Director</span>
+                        <h3 class="h2 bureau-bio_name">Steve Antonakes</h3>
+                        <a class="jump-link jump-link__right" href="about-steve-antonakes">
+                            Antonakes's bio
+                        </a>
+                    </div>
+                </div>
+                <a class="jump-link jump-link__right" href="/the-bureau/bureau-structure/">
+                    Organizational structure and leadership
+                </a>
+            </section>
+        </div>
+        <div class="content-l_col-1-2">
+            <section class="block
+                        block__border-top
+                        block__padded-top
+                        block__flush-bottom">
+                <h2>Strategy & Budget</h2>
+                <p>
+                    We work for the American public.
```

Comment:
I think it's 

```
We work for the American public.
We publish our strategy and budget
so you can understand our priorities.
```

Any chance we could hop into a Google chat hangout to hash out what needs to be done for this PR?